packages/abstract-sdk/src/apis/version_registry.rs

```diff
@@ -80,4 +100,14 @@ impl<'a, T: ModuleRegistryInterface> ModuleRegistry<'a, T> {
         )?)?;
         Ok(modules.swap_remove(0))
     }
+
+    /// Smart query for a namespace
```

Comment:
What does this return? Important to know that this gets the account related to the namespace

Reply:
Corrected